src/modules/Providers/dynamic-plugin.ts

```diff
@@ -69,4 +72,13 @@ export const extensions: EncodedExtension[] = [
       },
     },
   } as EncodedExtension<ActionProvider>,
+
+  {
+    type: 'console.model-metadata',
```

Comment:
hi, sorry for the unrelated code, this changes the provider link render from default light-blue "P" to dark blue "PR"